templates/.gitignore

```diff
@@ -0,0 +1,5 @@
+.DS_Store
+node_modules
+*-lock.*
+*.lock
+*.log
```

Comment:
Can we just move these to the top level .gitignore, if they are not there already? (Ensuring we don't ignore our top level package-lock.json!)

Reply:
Moved the lock related ones and should only affect the locks in templates.